src/master-playlist-controller.js

```diff
@@ -505,7 +532,7 @@ export default class MasterPlaylistController extends videojs.EventTarget {
    * making it unavailable for selection by the rendition selection algorithm
    * and then forces a new playlist (rendition) selection.
    */
-  blacklistCurrentPlaylist_(error) {
+  blacklistCurrentPlaylist(error = {}) {
```

Comment:
This method name is very confusing. blacklistCurrentPlaylist() optionally takes a non-current playlist to blacklist? Ugh.